### DISCUSSION TITLE
3dtiles新增模型压平功能

### DISCUSSION THREAD

**Discussion by zhonghua929** [zouyaoji/vue-cesium#606] [Ideas]
你好！ 有个新需求3dtiles添加后和gltf模型组合使用，重叠部分就显示冲突，希望通过CustomShader压平3dtiles模型，或者剪切重叠的部分，期待能在VcPrimitiveTileset基础上新增压平和剪切功能，谢谢！

**Comment by zouyaoji** [comment]
目前没有时间精力搞这个，希望有人能贡献。
